src/app/components/PageLayout/Header.tsx

```diff
@@ -49,7 +50,7 @@ export const Header: FC = () => {
               showText={!scrollTrigger && !isMobile}
             />
           </Grid>
-          {scope && (
+          {scope && isScopeSelectorNeeded(scope) && (
```

Comment:
Not a fan of function calls inside JSX.

Reply:
Would you prefer having it declared as a const, before the return command?

Reply:
never mind, moved to a const before the JSX part.